src/autocompletion_provider/modelAutocompletionProvider.ts

```diff
@@ -104,15 +105,22 @@ export class ModelAutocompletionProvider
       const models = added.nodeMetaMap.entries();
       this.modelAutocompleteMap.set(
         added.project.projectRoot.fsPath,
-        Array.from(models).map(([key, model]) => ({
-          label: `(${model.package_name}) ${key}`,
-          insertText:
-            model.package_name === projectName
-              ? key
-              : `${model.package_name}, ${key}`,
-          kind: CompletionItemKind.Value,
-          detail: "Model",
-        })),
+        Array.from(models)
+          .filter(
+            ([key, model]) =>
+              !model.uniqueId.startsWith(
```

Comment:
Please use `model.resource_type === DBTProject.RESOURCE_TYPE_ANALYSIS` instead of relying on uniqueid

Reply:
@AdiGajbhiye : Maybe also take that into account in other places.

Reply:
right now we don't have `resource_type` in model [ref](https://github.com/AltimateAI/vscode-dbt-power-user/blob/master/src/domain.ts#L18-L30)

should we add it in this PR?

Reply:
Yeah it is better than relying on the uniqueId